app/views/core/LoadingProgress.vue

```diff
@@ -56,6 +56,16 @@
           const statuses = this.statuses.map((status) => {
             if (typeof status === 'boolean') {
               return status ? 0 : 100
+            } else if (typeof status === 'object' && status.length > 0) {
```

Comment:
Would this be clearer with the [`isArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) on the Array prototype?

`length` on an object returns undefined so an object wouldn't pass the test.